empiric-package/empiric/core/mixins/oracle.py

```diff
@@ -42,30 +42,36 @@ async def publish_spot_entry(
         return invocation
 
     async def publish_many(
-        self, entries: List[Entry], pagination=0, max_fee=int(1e16)
-    ) -> InvokeResult:
+        self, entries: List[Entry], pagination=0, max_fee=int(1e18)
```

Comment:
Think I would prefer the default of `max_fee=None` and then using `estimate_fee` of the underlying Starknet.py package

Reply:
are you sure that's how it works? https://github.com/software-mansion/starknet.py/blob/5742478e1708f8b44fcc7de70ce0a34b8f428fdd/starknet_py/net/account/account_client.py#L440

Reply:
Also, https://starknetpy.readthedocs.io/en/latest/guide.html?highlight=estimate_fee#fees

Reply:
https://github.com/software-mansion/starknet.py/blob/5742478e1708f8b44fcc7de70ce0a34b8f428fdd/starknet_py/net/gateway_client.py#L191

Reply:
Let's fix this in a follow-up PR. Agree that the current max defaults are high enough that we will never hit them in the short term